src/utils/codec.ts

```diff
@@ -69,11 +97,50 @@ export const BASE64: CodecScheme = {
 }
 
 /**
- * Unicode code points range: `0 - 0x10FFFF`.
+ * BASE1024 Encoding
+ */
+export const BASE1024: CodecScheme = {
+    bits: 10,
+    name: 'Base1024',
+    table: BASE_1024,
+    tail: '=',
+}
+
+/**
+ * BASE2048 Encoding
+ */
+export const BASE2048: CodecScheme = {
+    bits: 11,
+    name: 'Base2048',
+    table: parseEmojis(BASE_2048),
+    tail: '=',
+}
+
+/**
+ * Assert `CodecSchema`
  *
- * If there is no element at point, the value is `undefined`.
+ * - `bits` : should be `number`
+ * - `table`: `length` of table should be `1 << bits`
+ * - `tail` : should be `string`
  */
-type Point = number | undefined
+function assertScheme(scheme: CodecScheme) {
```

Comment:
```suggestion
function assertScheme(scheme: any): asserts scheme is CodecScheme {
```

see https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html